device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/MultiplexingClient.java

```diff
@@ -349,8 +356,12 @@ public void registerDeviceClient(DeviceClient deviceClient, long timeoutMillisec
      * <p>
      * @throws InterruptedException If the thread gets interrupted while waiting for the registration to succeed. This
      * will never be thrown if the multiplexing client is not open yet.
-     * @throws MultiplexingClientDeviceRegistrationAuthenticationException If one or more devices failed to register. Details for each failure can be found
-     * in this exception. Any devices not found in the map of registration exceptions provided by this exception have registered successfully.
+     * @throws MultiplexingClientDeviceRegistrationAuthenticationException If one or more devices failed to register.
```

Comment:
fullstop to comma

Reply:
I'm confused, are you suggesting it should read "If one or more devices failed to register, details for each failure can be found in the map provided by..."? The period seems appropriate here to me

Reply:
Generally, the first line of this exception javadoc explains when it is throw, so "If one or more devices failed to register." seems appropriate as is